client/components/AddItem/Header.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { Times } from 'styled-icons/fa-solid';
+import BlueTimes from './BlueTimes';
```

Comment:
out of curiosity, what is this doing?

Reply:
Times is an SVG provided by the styled-icons library. BlueTimes is a custom component where I just changed the color to blue. I was using it in two different places so I made it its own file. I also had to mock it in Jest because Jest fails when trying to read the styled-icons library.